internal/services/containerapps/helpers/container_apps.go

```diff
@@ -1335,6 +1549,13 @@ func ContainerEnvVarSchemaComputed() *pluginsdk.Schema {
 	}
 }
 
+func expandInitContainerEnvVar(input BaseContainer) *[]containerapps.EnvironmentVar {
+	container := Container{
+		Env: input.Env,
+	}
+	return expandContainerEnvVar(container)
+}
+
```

Comment:
Can we perform this fully, rather than this conversion into another model:
```suggestion
func expandInitContainerEnvVar(input BaseContainer) *[]containerapps.EnvironmentVar {
	envs := make([]containerapps.EnvironmentVar, 0)
	if input.Env == nil || len(input.Env) == 0 {
		return &envs
	}

	for _, v := range input.Env {
		env := containerapps.EnvironmentVar{
			Name: pointer.To(v.Name),
		}
		if v.SecretReference != "" {
			env.SecretRef = pointer.To(v.SecretReference)
		} else {
			env.Value = pointer.To(v.Value)
		}

		envs = append(envs, env)
	}
	
	return &envs
}

```